src/Life/Github.hs

```diff
@@ -61,6 +63,31 @@ insideRepo action = do
 pushRepo :: Text -> IO ()
 pushRepo = insideRepo . pushka
 
+-- | Returns true if local @dotfiles@ repository is synchronized with remote repo.
+checkRemoteSync :: IO Bool
+checkRemoteSync = do
+    "git" ["fetch", "origin", "master"]
+    localHash  <- "git" $| ["rev-parse", "master"]
+    remoteHash <- "git" $| ["rev-parse", "origin/master"]
+    pure $ localHash == remoteHash
+
+withSynced :: IO a -> IO a
+withSynced action = insideRepo $ do
+    infoMessage "Checking if repo is synchnorized..."
+    isSynced <- checkRemoteSync
+    if isSynced then do
+        infoMessage "Repo is up-to-date"
+        action
+    else do
+        warningMessage "Local version of repository is old"
+        shouldSync <- chooseYesNo "Do you want to sync repo with remote?"
+        if shouldSync then do
+            "git" ["pull"]  -- TODO: some other command?
```

Comment:
This command takes some time. But since I already did `git fetch` it can be implemented faster as `git rebase origin/master`. Should we use this way?

Reply:
Yes, I agree that `rebase` it better here.